src/amo/components/Header/index.js

```diff
@@ -68,13 +68,7 @@ export class HeaderBase extends React.Component {
     );
 
     const viewProfileURL = siteUser ? `/user/${siteUser.username}/` : null;
-    const viewProfileLinkProps = _config.get('enableUserProfile')
-      ? { to: viewProfileURL }
-      : { href: viewProfileURL };
     const editProfileURL = siteUser ? '/users/edit' : null;
```

Comment:
can this be inlined since only used once?

Reply:
ah sure